doc/ipython-notebooks/link_checker.py

```diff
@@ -0,0 +1,41 @@
+import json
+import re
+import subprocess
+import sys
+#requirement is python >= 3.5
```

Comment:
i think we only check for python3 in cmake, so if you require 3.5 then you should add a version check here and raise an error.

Reply:
Will add that as well in the latest commit. Requests module does require python>=3.5
Will make a function for checking python version requirement.

Reply:
I mean you can just use [urllib](https://docs.python.org/3/library/urllib.request.html#module-urllib.request) which is just part of python.

Reply:
Oh. sorry for this.Didn't knew about urllib. That makes things much more easier.